### ISSUE TITLE
Crash when retrieving network capabilities on android 11

### ISSUE THREAD

**Issue by lcdcinq** [nextcloud/android#9849]
************ CAUSE OF ERROR ************
```
java.lang.SecurityException: Package android does not belong to 10259
	at android.os.Parcel.createExceptionOrNull(Parcel.java:2376)
	at android.os.Parcel.createException(Parcel.java:2360)
	at android.os.Parcel.readException(Parcel.java:2343)
	at android.os.Parcel.readException(Parcel.java:2285)
	at android.net.IConnectivityManager$Stub$Proxy.getNetworkCapabilities(IConnectivityManager.java:2456)
	at android.net.ConnectivityManager.getNetworkCapabilities(ConnectivityManager.java:1410)
	at com.nextcloud.client.network.ConnectivityServiceImpl.getConnectivity(ConnectivityServiceImpl.java:118)
	at com.owncloud.android.utils.FilesSyncHelper.lambda$restartJobsIfNeeded$0(FilesSyncHelper.java:207)
	at com.owncloud.android.utils.-$$Lambda$FilesSyncHelper$78U1WN_nzr5HiGIrRZMThSEamz8.run(Unknown Source:10)
	at java.lang.Thread.run(Thread.java:923)
Caused by: android.os.RemoteException: Remote stack trace:
	at android.app.AppOpsManager.checkPackage(AppOpsManager.java:7794)
	at com.android.server.ConnectivityService.getNetworkCapabilities(ConnectivityService.java:1779)
	at android.net.IConnectivityManager$Stub.onTransact(IConnectivityManager.java:978)
	at android.os.Binder.execTransactInternal(Binder.java:1154)
	at android.os.Binder.execTransact(Binder.java:1123)
```

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30150090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Redmi
Device: sunny
Model: M2101K7AG
Id: RKQ1.201022.002
Product: sunny_global

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: V12.5.5.0.RKGMIXM

### Steps to reproduce
1. It occures whitout reason
2. 
3. 

### Expected behaviour
- nothing should happen, thé app run in background

### Actual behaviour
- sometimes thé app hangs

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AlvaroBrey** [comment]
Is this app on a work profile or something like that?

**Comment by AlvaroBrey** [comment]
known bug on android 11, ref: https://github.com/google/ExoPlayer/issues/9002

**Comment by joshtrichards** [comment]
Fixed upstream in https://github.com/google/ExoPlayer/commit/2e21208f639c7eb5baf5e7a3ae1180b13be3b733

Fixed in #9206 (when we bumped to exoplayer v2.16.0).
Released in v3.19.0
